relaton/serializers/bibxml/reference.py

```diff
@@ -89,7 +89,12 @@ def create_reference(item: BibliographicItem) -> Element:
     if len(abstracts) > 0:
         front.append(create_abstract(abstracts))
 
-    ref = E.reference(front)
+    # refcontent
+    refcontent = create_refcontent(item.extent)
```

Comment:
Until now, all `create_*` functions under `serializers.bibxml` returned an `Element`. It’s a naming convention that was violated. I refactored `create_refcontent()` to return an `Element`.

Reply:
Renamed to `build_refcontent_string()`. I think it’s better to return an `str`, to simplify testing.